Dockerfiles/alpine/Dockerfile

```diff
@@ -0,0 +1,8 @@
+ARG GIT_DESC=undefined
+FROM yarmak/dumbproxy:${GIT_DESC} as base
```

Comment:
Maybe we should not reuse built image explicitly, but instead just rely on caching of docker layers? Then we will not have cross dependency of build jobs and we will be able to copy stage1 from already existing dockerfile as is.

Reply:
Made it within one `Dockerfile` and using `DOCKER_BUILDKIT=1`. Please check it.

Reply:
Cool! I never knew such thing is possible. Let's see how it works!